build.sbt

```diff
@@ -2,13 +2,13 @@ name := "persist-json"
 
 organization := "com.persist"
 
-version := "1.1.3"
+version := "1.2.0-WP"
 
 scalaVersion := "2.11.7"
 
 scalacOptions ++= Seq("-deprecation", "-unchecked")
 
-viewSettings
+//viewSettings
```

Comment:
Should we just remove this? Commented out code is dregs. We can always revert/add back if necessary.